### DISCUSSION TITLE
Welcome to SshKeyMan Discussions!

### DISCUSSION THREAD

**Discussion by Bandeapart1964** [catpuppyapp/SshKeyMan#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pondswimmers** [comment]
completely green and ignorant, but humble enough to ask. Will someone with slight patience explain ssh, and Lan , ect. Or a link to learn ." Ive arrived in the deep end with floaties on ";- Well Im here and smart enough to have brought floaties.- That's gotta get me five mins of someones time lol 😂

**Reply by Bandeapart1964** [reply]
hi, you can ask chatgpt😁

**Comment by Pondswimmers** [comment]
thanks ! never thought of that 🧐

**Reply by Bandeapart1964** [reply]
🤣, the chatgpt even smarter than me, maybe
